examples/plugins/inspec-resource-lister/test/unit/README.md

```diff
@@ -0,0 +1,17 @@
+# Unit Testing Area for Example Plugins
+
+## What example tests are provided?
+
+Here, since this is a CliCommand plugin, we provide two sets of unit tests:
+
+ * plugin_def_test.rb - Would be useful in any plugin.  Verifies that the plugin is properly detected and registered.
+ * cli_args_test.rb - verifies that the expected command are present, and that they have the expected options and args.
```

Comment:
Looks like you capitalize after the hyphen in the sentence above, could we here too? Also, should be `commands are`.